roles/openshift_master/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
       api_port: "{{ openshift_master_api_port | default(None) }}"
       api_url: "{{ openshift_master_api_url | default(None) }}"
       api_use_ssl: "{{ openshift_master_api_use_ssl | default(None) }}"
-      public_api_url: "{{ openshift_master_public_api_url | default(None) }}"
+      public_api_url: "{{ openshift_master_public_api_url | default('https://' ~ openshift.common.public_ip ~ ':8443') }}"
```

Comment:
Thanks, for future reference, if you push to your same branch that you created the pull request from it will update the pull request. If you've rebased you can use a force push `-f` to allow for a non fast-forward push. That way all the conversation and work will remain in one pull request.